tests/truncate-applayer-test/test.yaml

```diff
@@ -0,0 +1,14 @@
+args:
+  - -k none --set stream.reassembly.depth=8kb
+
+checks:
+- filter:
+    count: 1
+    match:
+      event_type: alert
+      alert.signature_id: 1
+- filter:
+    count: 0    # because of the depth constraints
```

Comment:
We could also have a second test without the `stream.reassembly.depth=8kb` to show that we get the alert